languages/csharp/AppEncryption/AppEncryption.IntegrationTests/Regression/AppEncryptionParameterizedTest.cs

```diff
@@ -140,5 +149,88 @@ public AppEncryptionParameterizedTest()
                     Times.Never);
             }
         }
+
+        private class AppEncryptionParameterizedTestData : IEnumerable<object[]>
+        {
+            private static readonly Random Random = new Random();
+            private readonly ConfigFixture configFixture;
+
+            public AppEncryptionParameterizedTestData()
+            {
+                configFixture = new ConfigFixture();
+                IMetricsRoot metrics = new MetricsBuilder()
+                    .Configuration.Configure(options => options.Enabled = false)
+                    .Build();
+
```

Comment:
nit: if a newline is used at all here, the metrics related lines should be the ones grouped together